Src/Plugins/Security/Security.csproj

```diff
@@ -5,7 +5,7 @@
   <Import Project="$(MSBuildThisFileDirectory)..\..\..\Targets\build.plugins.props" />
 
   <PropertyGroup Label="AssemblyAttributes">
-    <TargetFrameworks>netstandard2.0;net472</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net472</TargetFrameworks>
```

Comment:
>1 [](http://example.com/codeflow?start=35&length=1)

Do you know why this plug is inconsistent with other MS compilation settings?

Reply:
we started with 2.0, but the other package got a higher dependency that required 2.1.

Reply:
In the security.tests test, can you change to netstandard2.1? (the strings in the unit tests)
that will probably fix the build issue

Reply:
fixed.